lua/refactoring/refactor/106.lua

```diff
@@ -1,62 +1,16 @@
-local ts_utils = require("nvim-treesitter.ts_utils")
-local Query = require("refactoring.query")
-local utils = require("refactoring.utils")
 local Pipeline = require("refactoring.pipeline")
-local selection_setup = require("refactoring.pipeline.selection_setup")
-local refactor_setup = require("refactoring.pipeline.refactor_setup")
-local get_input = require("refactoring.get_input")
-local create_file = require("refactoring.pipeline.create_file")
-local post_refactor = require("refactoring.pipeline.post_refactor")
+local selection_setup = require("refactoring.tasks.selection_setup")
+local refactor_setup = require("refactoring.tasks.refactor_setup")
+local create_file = require("refactoring.tasks.create_file")
+local post_refactor = require("refactoring.tasks.post_refactor")
 local get_selected_local_defs = require(
-    "refactoring.pipeline.get_selected_local_defs"
+    "refactoring.tasks.get_selected_local_defs"
 )
+local extract_tasks = require("refactoring.tasks.extract_method")
 local Config = require("refactoring.config")
 
 local M = {}
 
-local function get_local_definitions(bufnr, local_defs, function_args)
```

Comment:
Lets leave this code for now.  

All actual refactoring I want to stay for a couple more (as I stated really crappily from my phone).

I want it to get a bit more painful and really think about the solution.  I don't necessarily just want to move everything to a new file.  My guess is that once I do another one of these tests that require scope capturing (split loop, which is next) it will naturally come out as I avoid code duplication.

Does that make sense?  So in other words just keep all the files the same except the rename.  Then we can tackle the moving of logic separately. 

Last I am open to you trying before the need arises if you feel like you have a good idea, just make it a separate pr.